pkg/processor/trigger/http/types.go

```diff
@@ -85,8 +85,8 @@ func createCORSConfiguration(CORSConfiguration *cors.CORS) *cors.CORS {
 		corsInstance.AllowMethods = CORSConfiguration.AllowMethods
 	}
 
-	if CORSConfiguration.AllowOrigin != "" {
-		corsInstance.AllowOrigin = CORSConfiguration.AllowOrigin
+	if len(CORSConfiguration.AllowOrigins) > 0 {
+		corsInstance.AllowOrigins = CORSConfiguration.AllowOrigins
```

Comment:
Why not always?

Reply:
not to override the defaults & to allow minimum configuration to make the CORS working
```yaml
cors:
  enabled: true
```
will simply use the defaults (for the allowOrigins would be ["*"])